src/components/SidebarFilter/SidebarFilter.tsx

```diff
@@ -68,6 +71,7 @@ export const SidebarFilter = ({
           ))}
         </div>
       )}
+      {children}
```

Comment:
I think this is a clearer solution than adding a new component (SidebarSearch) and it all props.

Reply:
Agreed